testsuite/integration/clustering/pom.xml

```diff
@@ -282,14 +282,14 @@
                         </dependencies>
                         <executions>
                             <execution>
-                                <phase>process-test-resources</phase>
+                                <phase>generate-test-resources</phase>
```

Comment:
Is this now enough for the Galleon layers execution profile? `process-*` woud have copied the content, IIUC [1], while `generate-*` (which for sure is appropriate for `-Dts.bootable`) generates it.
WDYT?

Reply:
I just moved the key pair generation one phase earlier. I suppose it in the correct phase now, since it _generates_ a key pair. The only impact should be that the resources are available at the bootable jar packaging time/phase (line 1382,  <extraServerContent>${wildfly.dir}</extraServerContent>).

Reply:
> The only impact should be that the resources are available at the bootable jar packaging time/phase

Which is what we want. Thanks!